src/main/java/core/basesyntax/model/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+public class Square extends Color {
```

Comment:
The class 'Square' should not extend 'Color'. In object-oriented design, a square 'is not' a color, but 'has a' color. You should use composition instead of inheritance here.